pkg/alertmanager/multitenant.go

```diff
@@ -175,6 +175,10 @@ func NewMultitenantAlertmanager(cfg *MultitenantAlertmanagerConfig, logger log.L
 		return nil, fmt.Errorf("unable to create Alertmanager data directory %q: %s", cfg.DataDir, err)
 	}
 
+	if cfg.ExternalURL.URL == nil {
+		return nil, fmt.Errorf("unable to create Alertmanager, no 'web.external-url' configured")
```

Comment:
Either we display the exact CLI flag (`-alertmanager.web.external-url`) or we explain it in plain english. I would discourage the CLI flag because the user may use the YAML config file and have no idea about the flag.

```suggestion
		return nil, fmt.Errorf("unable to create Alertmanager because the external URL has not been configured")
```